docs/advanced_usage.rst

```diff
@@ -517,6 +517,7 @@ uWSGI
 
 - Threads must be enabled with the `enable-threads <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#enable-threads>`__ or `threads <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#threads>`__ options.
 - Lazy apps must be enabled with the `lazy-apps <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#lazy-apps>`__ option.
+- Skip atexit, `--skip-atexit <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#skip-atexit>`__, must be enabled when `lazy-apps <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#lazy-apps>`__ is enabled. This is to avoid crashes from native extensions that could occur during at exit teardown.
```

Comment:
From our offline discussions, it seems like this is a known issue that's not exclusive to our library. Do you have a link to a GitHub issue or something else that explains it more? If so, do you think it'd be worth referencing it here? Like, "this is a known issue with uwsgi. See <example>"